ayusshrathore/game-hub
# Background Color Updated To Enhance The Appearance in Light Mode.

## Pull Request Title

Background Color of the gameCard In Light Mode Updated.

## Description
Background Color of the gameCard component in Light Mode changed from white to ghost white to differentiate gameCard from the main Body.

## Changes Made

Changes made in GameCard.tsx file. Changed the background color of the component from white to ghost-white in light mode to differentiate it from the main body. 

## Screenshots 
# Image Before Changes
![imagebeforechanges](https://github.com/ayusshrathore/game-hub/assets/68887867/f32cae52-28dc-4246-939c-84104c20c0ea)

# Image After Changes
![imageafterchanges](https://github.com/ayusshrathore/game-hub/assets/68887867/54d89de7-c340-4207-b8e9-0835affe9aa4)

## How to Test

To test this pull request, follow these steps:

1. Visit the page or component where the background color change was applied(Main Page in light mode).

2. Observe the background color. It should now be ghost-white.

3. Scroll and interact with the page/component to ensure that the background color change does not cause any unintended side effects or issues.

4. Compare the background color with the previous version to verify the change.

That's it! If you can confirm that the background color has been updated as intended without any adverse effects, this pull request is ready to be merged.

## Closing Note

I want to express my appreciation for taking the time to review this pull request. Your feedback and insights are valuable to me. Please share your feedback with me.
